<ui version="4.0" >
 <author>(C) 2002, 2003 Andras Mantia &lt;amantia@kde.org></author>
 <comment>***************************************************************************
 *                                                                         *
 *   This program is free software; you can redistribute it and/or modify  *
 *   it under the terms of the GNU General Public License as published by  *
 *   the Free Software Foundation; version 2 of the License.               *
 *                                                                         *
 ***************************************************************************
</comment>
 <exportmacro></exportmacro>
 <class>DTDSelectDialog</class>
 <widget class="QWidget" name="DTDSelectDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>161</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>440</width>
    <height>160</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>DTD Selector</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>11</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="0" column="0" colspan="2" >
    <widget class="QLabel" name="messageLabel" >
     <property name="text" >
      <string>Dialog message:
Dialog message2:</string>
     </property>
     <property name="scaledContents" >
      <bool>true</bool>
     </property>
     <property name="alignment" >
      <set>Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="QComboBox" name="dtdCombo" />
   </item>
   <item row="2" column="0" >
    <widget class="QLabel" name="textLabel" >
     <property name="text" >
      <string>Current DTD:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="comboLabel" >
     <property name="text" >
      <string>Select DTD:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" >
    <widget class="QLineEdit" name="currentDTD" >
     <property name="readOnly" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2" >
    <widget class="QCheckBox" name="convertDTD" >
     <property name="text" >
      <string>Conver&amp;t the document to the selected DTD</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2" >
    <widget class="QCheckBox" name="useClosestMatching" >
     <property name="text" >
      <string>Do not show this dialog, use the closest matching DTD</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <tabstops>
  <tabstop>dtdCombo</tabstop>
  <tabstop>currentDTD</tabstop>
  <tabstop>convertDTD</tabstop>
  <tabstop>useClosestMatching</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
